docs/index.md

```diff
@@ -68,16 +67,15 @@ resource "hcp_aws_network_peering" "example_peering" {
 
 // Create a Consul cluster in the same region and cloud provider as the HVN
 resource "hcp_consul_cluster" "example" {
-  hvn_id         = hcp_hvn.example_hvn.hvn_id
-  cluster_id     = "hcp-tf-example-consul-cluster"
-  cloud_provider = var.cloud_provider
-  region         = var.region
+  hvn_id     = hcp_hvn.example_hvn.hvn_id
+  cluster_id = "hcp-tf-example-consul-cluster"
+  tier       = "development"
 }
 ```
 
 ## Schema
 
-### Required
+### Optional
```

Comment:
Hmm, was this changed by the docs generator? I agree with the change, but wondering if we should also change these 2 fields in the scheme to be optional: https://github.com/hashicorp/terraform-provider-hcp/blob/main/internal/provider/provider.go#L37

Reply:
Oh good point. I wonder why this changed if it's still required in the provider 🤔